src/index.html

```diff
@@ -13,19 +13,35 @@
     <title>Document</title>
     <link
       rel="stylesheet"
-      href="style.css"
+      href="styles/style.css"
```

Comment:
The attributes for the `link` tag should start on a new line with 2-space indentation related to the tag, as per the checklist for keeping attributes correctly formatted.